src/js/views/maps/viewfinder/ViewfinderView.js

```diff
@@ -66,7 +69,11 @@ define(
        */
       initialize({ model: mapModel }) {
         this.viewfinderModel = new ViewfinderModel({ mapModel });
-        this.panelsModel = new ExpansionPanelsModel();
+        this.panelsModel = new ExpansionPanelsModel({ isMulti: true });
+      },
+
+      getZoomPresets() {
```

Comment:
```suggestion
      /**
       * Get the ZoomPresetsView element.
       * @returns {JQuery} The ZoomPresetsView element.
       * @since x.x.x
       */
      getZoomPresets() {
```